images/nfs/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN dnf install -y tar gcc cmake autoconf libtool bison flex make gcc-c++ krb5-d
  && cd nfs-ganesha-2.4.1 \
  && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_CONFIG=vfs_only src/ \
  && patch -p1 < /nfs-ganesha.patch \
- && make \
+ && make -j \
```

Comment:
Remove the empty `-j` flag, we don't want unlimited makefile jobs to run in each CI run.

Reply:
OK. I have removed it.